.github/workflows/tests.yml

```diff
@@ -27,30 +24,15 @@ jobs:
 
     - name: Set up Homebrew
       run: |
+        cd $(brew --repo)
```

Comment:
Could this be linked to the `$PWD` instead of needing to run it in multiple steps?

Reply:
(not blocking)